mk/config.mk

```diff
@@ -648,6 +648,9 @@ endif
 # Enables backwards compatible derivation of RPMB and SSK keys
 CFG_CORE_HUK_SUBKEY_COMPAT ?= y
 
+# Use SoC specific tee_otp_get_die_id implementation for SSK key generation
+#CFG_CORE_HUK_SUBKEY_COMPAT_USE_OTP_DIE_ID ?= y
```

Comment:
Please add a comment that this depends on `CFG_CORE_HUK_SUBKEY_COMPAT` to be enabled like you describe it in the commit message.
Leave this as
`CFG_CORE_HUK_SUBKEY_COMPAT_USE_OTP_DIE_ID ?= n`
without a `#` in front.